packages/presentation/src/components/message/Nodes.svelte

```diff
@@ -63,8 +63,19 @@
     {:else if node.nodeName === 'DIV'}
       <div><svelte:self nodes={node.childNodes} /></div>
     {:else if node.nodeName === 'A'}
-      <a href={node.getAttribute('href')} target={node.getAttribute('target')}
-        ><svelte:self nodes={node.childNodes} /></a
+      <a
+        href={node.getAttribute('href')}
+        target={node.getAttribute('target')}
+        on:click={(e) => {
```

Comment:
I think we need to check why we handle page new url and reload instead of update in Workbench itself.

Reply:
It's related to the `target` property, which is `_blank` by default for links created by tiptap, i.e. open in new tab
If we remove that prop, then links will be opened in the same tab but with reload, this is the default behavior of `<a>` tag
That's why we handle click and prevent default behavior if needed